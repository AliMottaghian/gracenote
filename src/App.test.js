import App from "./App";

describe('<App />',()=>{
  const wrapper =  shallow(<App />);
  it("should render App component without crashing",()=>{
    expect(wrapper).toMatchSnapshot()
  })
})