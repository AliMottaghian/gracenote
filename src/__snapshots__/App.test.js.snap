// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render App component without crashing 1`] = `
<div
  className="App"
>
  <Header />
  <Routes />
</div>
`;
